workflow-ui/core-common/src/main/java/com/squareup/workflow1/ui/Container.kt

```diff
@@ -0,0 +1,46 @@
+package com.squareup.workflow1.ui
+
+import com.squareup.workflow1.ui.Compatible.Companion.keyFor
+
+/**
+ * A model type comprised of a set of other models.
```

Comment:
I sometimes get thrown off when you call these `model`s. Why model when we use `Rendering` elsewhere for the general type and `Screen` for the marker interface?

I get that this concept (and its generics) are generalized enough to be used for anything but this is opinionated and designed for use with Workflow UI Renderings, no?

Reply:
It's my recent forays into Compose leaking in. What we call "renderings" they generally refer to as "models." I like it better as a general term, but you're right: this is Workflow UI land, and "rendering" is entrenched.